### ISSUE TITLE
Ready-to Print Output

### ISSUE THREAD

**Issue by efspeck** [chonger/cueballer#5]
Just want to get into writing something we've all discussed and link a doc where we can all access it.

We currently have a three step user process:

Step 1: Obtain and correctly format a script.

Step 2: Copy the script and paste into cueballer, click the button, and do a happy dance.

Step 3: Copy cueballer output and paste into word processing software and reformat for print.

I'll leave automating step 1 for another thread (perhaps the thorniest issue left?), but [linked ](https://docs.google.com/document/d/1UcESJ5m9LfeXbp801i1vkUfUwaWBfPzPLQ7MgcFuf9k/edit?usp=sharing) is my suggested goal for step 3 if we were able to automate it.

This is the best format I've found so far in terms of margins and font size, balancing between readability and keeping verse lines intact, but Andrew, if you have any thoughts based on your experience this week re-printing the 12th night scripts please let us know.  Two interesting notes:  1) in the interest of making the font as large as possible, I'm making the margins as small as possible, but margins were actually really important to the original parts, as it's where actors would write in their own stage directions. 2) I'm using 8.5" x 11" as its obviously the most likely to be in a user's printer, but I prefer legal for my own as it reduces the number of page breaks and thus taped joints and thus rolls better.  Some nerdy bonus features down the line could involve delving into Tiffany Stern's "Shakespeare in Parts" to make the most historically accurate margin and page sizes, getting the stage directions in the margins and getting a "correct" line per page count.  There's also got to be a spool printer out their similar to a receipt printer that would print with zero page breaks, but with no historical or research value and minimal user value, I'm not terribly interested in finding one.  Maybe someday we'll have that machine at a kiosk at Shakespeare's Globe to sell parts to tourists, but until then...

Finally, as a github newb, is this the best way for me to be communicating this?  Is this an "issue" or is there a more appropriate place for my ramblings?

**Comment by chonger** [comment]
i think issues are a good way to keep organized records of conversations.

re: the Issue at hand - i think that your summary of the 3 step procedure is accurate, and that for our next pass on the system we should decide exactly what we want to do for step 3 (the topic of this issue, right?)

The big question is - do we want to do it at all?  The workflow where you paste into Word gives you the director total control over the formatting.  Anything that we would build into the website itself would most likely make assumptions about these things (margins, fonts, etc) as it would be overkill to build an entirely new word processor.

That said, I think it would be great to figure out some bare minimum improvement so that you could still copy paste into a Word Doc, but also you could just take some sensible defaults and print directly from cueballer the webpage.  This would probably involve a "print" button to be built into cueballer and also something to do around formatting.  This would be a totally new bit to the system, Andrew if you're interested it might be a good bit for you to own dev on?  

So to summarize, this is what id suggest:

1) we decide if we want to do this at all - the added value is time savings to the director, and the risk is loss of control over formatting.  This is up to you guys to decide.

2) if we're going to do something, lets get a simple easy to check off bullet point list of what those things are.  This doesn't need to be the end all of the features, but it should be ones they you're confident enough would be almost certainly desired by all directors.

**Comment by efspeck** [comment]
For the short term I might vote against doing anything.

Long term, if there is enough interest to justify the extra work, I think the ideal is a "PRINT" button that automatically saves the output text as a two columned PDF.  With know idea of what work that would require, I'd say cueballer has accomplished what it needs to to be of real value, and a printable output is icing.
